pull-readmes: Make relative file links absolute

ochosi opened the pull request:
~~I'm sorry I'm sorry I'm sorry.~~
I'm actually a little less sorry now.

dvdhrm commented on _collections/_news/2020-10-06-release-cockpit-composer-25.md:
```diff
@@ -0,0 +1,27 @@
+categories: [fedoraplanet]
+---
+
+We are happy to announce the release of cockpit-composer 25. This release has 
+no major features but contains useful fixes.
```
You are the native speaker here, so I will refrain from being too picky ;) But shouldn't this read `This release has no major **new** features`? Feels weird to say our release has no features!

Secondly, aren't all fixes `useful fixes`?

And because I think this is all very nit-picky, I will still merge it straight away ;)

ochosi committed:
pull-readmes: Make relative file links absolute

Convert the ugly shell script to a less ugly Python script.

ochosi committed:
developer-guide: Fix relative links

These fixes were done with the `pull_readmes.py` script and checked in
manually to unblock CI.
In the future, this link transformation will happen automatically as
part of the CI workflow.

ochosi committed:
pull-readmes: Add tests for script

schuellerf reviewed (approved):
LGTM
